2020/summer_meeting/session-2-culture.md

```diff
@@ -0,0 +1,104 @@
+# Session 2 - Culture
+Notetakers: James Griffin, Nikitas Tampakis
+
+## What cultural differences are we experiencing in our institutions and what challenges do they present?
+### Resistance changing the way one works
+ - Being unwilling to automate isn’t necessarily the challenge, but there is resistance often in letting go of old ways and where the direction to change comes from. ie peers vs manager.
+ - Some find that there is an unwillingness to change because a person might be uncomfortable with the new technologies.
+
+### Wanting to contribute, but lacking adequate knowledge or access
+ - In a smaller institution, the backup is a small group of other librarians without technical experience. How do you bring others up to speed?
+ - People with experience around a problem space, say working with Ansible for server provisioning, however, Ansible itself must be updated. What is the life cycle for this type of work?
+ - There are cases where staff members want to contribute to a solution, but they may not have the technical background.
+ - If one is hosting customized installations of ArchivesSpaces, with staff who want shell access to the servers. They cannot be granted root privileges.
+ - On-call idea, operations staff are often on the hook for work that isn’t quite an incident response but is an issue that falls into a gap.
+ - Challenges for team organization for vendor product support and applications you manage that integrate with vendor solutions. 
+   - There is a perception that you have more control over it than you do because it is possible to provide some JavaScript or other surface customization atop of the proprietary code base.
+   - When addressing the vendor product customization and bug fixes, the workflow changes.
+   -  There was recently a request to see what changes are being introduced to the vendor-produced system. While there are support tickets which were submitted, this and the internal documentation simply isn’t sufficient to communicate to stakeholders a list of change.
+   -  Someone from the web team might be borrowed for improving the UI, while members from other teams might address code cleaning. However, there is no dedicated team for this specific vendor product.
+
+### Accountability beyond the individual
+ - Workflow where Jenkins is running on pull requests, pull requests shouldn’t be merged unless tests pass. However people go ahead and merge the code if their new tests don’t pass. How do you help create a culture where people think beyond their immediate change, or failures that are outside of their own code?
```

Comment:
"if their new tests don't pass" I think is backwards. It should be "if failing tests are unrelated to changes introduced in the PR" or something like that.

Reply:
Thanks for catching that @hackmastera. The notes have been corrected.